pkgs/os-specific/darwin/swift-corelibs/corefoundation.nix

```diff
@@ -111,4 +112,6 @@ stdenv.mkDerivation {
       ln -s Versions/Current/$i $base/$i
     done
   '';
-}
+} // lib.optionalAttrs withRpathHook {
+  setupHook = ./corefoundation-setup-hook.sh;
```

Comment:
As alluded to in the eval failure log, looks like you forgot to commit `corefoundation-setup-hook.sh` :)

Reply:
Thanks! :)